Library/Homebrew/extend/os/linux/extend/ENV/std.rb

```diff
@@ -2,10 +2,18 @@
 # frozen_string_literal: true
 
 module Stdenv
-  def setup_build_environment(formula: nil, cc: nil, build_bottle: false, bottle_arch: nil, testing_formula: false)
+  def setup_build_environment(
+    formula: nil,
+    cc: nil,
+    build_bottle: false,
+    bottle_arch: nil,
+    testing_formula: false,
+    debug_symbols: false
+  )
     generic_setup_build_environment(
       formula: formula, cc: cc, build_bottle: build_bottle,
-      bottle_arch: bottle_arch, testing_formula: testing_formula
+      bottle_arch: bottle_arch, testing_formula: testing_formula,
+      debug_symbols: debug_symbols
```

Comment:
Feels weird to pass this through on Linux if it won't work/do anything (or will it)?

Reply:
Does it need to be passed for generica API reasons, or is it fine to drop the argument here? (It's called 'generic', so that made me want to give it a consistent API).

That being said, the argument is being used in super and I think this is expected to be a sub class so I think it should stay?

Reply:
Ok, maybe just worth adding a Ruby comment explaining this is a no-op but here for API consistency?

Reply:
So `debug_symbols` is only used in the `super` env on Mac and Linux. In `std` env on both, this is a no-op. I'm not sure if this is correct since I don't fully understand the interplay between super and std envs. We could remove for all std envs, or make the comment for all 3 std env methods. What do you think?